sdk/src/metrics/aggregation/histogram_aggregation.cc

```diff
@@ -176,6 +177,7 @@ std::unique_ptr<Aggregation> DoubleHistogramAggregation::Diff(
 
 PointType DoubleHistogramAggregation::ToPoint() const noexcept
 {
+  const std::lock_guard<opentelemetry::common::SpinLockMutex> locked(lock_);
```

Comment:
Wondering what kind of race conditino is this mutex protecting?

Reply:
both reads and writes should be protected. It can happen that while we are reading, some other thread writes to this filed.